### PR TITLE
Cleanup toyota tuning

### PR DESCRIPTION
**Pull request by haraschax**

### CODE REVIEW

File: selfdrive/car/toyota/tunes.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+from selfdrive.car.toyota.values import MIN_ACC_SPEED, PEDAL_HYST_GAP
+
+
+
+###### LONG ######
+def set_long_tune(tune, name):
+  if name == 'PEDAL':
```
Comment (adeebshihadeh) [inline selfdrive/car/toyota/tunes.py:8]: use an enum for the tunes https://docs.python.org/3/library/enum.html

File: selfdrive/car/toyota/tunes.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+from selfdrive.car.toyota.values import MIN_ACC_SPEED, PEDAL_HYST_GAP
+
+
+
+###### LONG ######
+def set_long_tune(tune, name):
```
Comment (adeebshihadeh) [inline selfdrive/car/toyota/tunes.py:7]: return a car.CarParams.LongitudinalPIDTuning
``` python
from cereal import car
tune = car.CarParams.LongitudinalPIDTuning.new_message()
```

File: selfdrive/car/toyota/interface.py [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
-from selfdrive.car.toyota.values import Ecu, CAR, TSS2_CAR, NO_DSU_CAR, MIN_ACC_SPEED, PEDAL_HYST_GAP, CarControllerParams
+from selfdrive.car.toyota.tunes import set_long_tune, set_lat_tune
+from selfdrive.car.toyota.values import Ecu, CAR, TSS2_CAR, NO_DSU_CAR, MIN_ACC_SPEED, CarControllerParams
 from selfdrive.car import STD_CARGO_KG, scale_rot_inertia, scale_tire_stiffness, gen_empty_fingerprint, get_safety_config
 from selfdrive.car.interfaces import CarInterfaceBase
 
 EventName = car.CarEvent.EventName
 
 
+
```
Comment (adeebshihadeh) [inline selfdrive/car/toyota/interface.py:12]: remove these extra lines

**Comment by adeebshihadeh** [comment]
I like this, it should add more friction to create new tunes. After merging this, we should do the same for at least Honda and Hyundai.

File: selfdrive/car/toyota/interface.py [hunk 4]
```diff
@@ -23,267 +24,212 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
...
+    if candidate in [CAR.LEXUS_CTH,]:
+      ret.safetyConfigs[0].safetyParam = 100
+    elif candidate in [CAR.LEXUS_IS,]:
+      ret.safetyConfigs[0].safetyParam = 77
+    elif candidate in [CAR.COROLLA,]:
+      ret.safetyConfigs[0].safetyParam = 88
+    elif candidate in [CAR.PRIUS,]:
+      ret.safetyConfigs[0].safetyParam = 66  # see conversion factor for STEER_TORQUE_EPS in dbc file
+    else:
+      ret.safetyConfigs[0].safetyParam = 73
```
Comment (adeebshihadeh) [inline selfdrive/car/toyota/interface.py:232]: doesn't really make sense to do this for the safetyParam. you could also just set a default of 73 on top and override for specific cars

Comment (HaraldSchafer) [reply selfdrive/car/toyota/interface.py]: Done

File: selfdrive/car/toyota/interface.py (hunk 3 above)
Comment (HaraldSchafer) [reply selfdrive/car/toyota/interface.py]: Done

File: selfdrive/car/toyota/tunes.py (hunk 1 above)
Comment (HaraldSchafer) [reply selfdrive/car/toyota/tunes.py]: Done

**Pull request merged**